### DISCUSSION TITLE
Perplexity Pro promo code

### DISCUSSION THREAD

**Discussion by gabri83** [community/community#143083] [Other Feature Feedback, Questions, & Ideas]
Anyone else had a problem redeeming the promo code? It says "Promo code not valid" when I try. I am not an existing pro customer

**Comment by dym-ok** [comment]
I've got the same issue here.

**Comment by yurtbayemre** [comment]
A friend was able to redeem the code about an hour ago, but unfortunately not me.

**Comment by benibits** [comment]
I have the same "Promo code invalid" issue. Looking at the code it doesn't look to be random generated if that works as a clue for investigating the issue

**Comment by knittingguy** [comment]
I have the same "Promo code invalid" error, too.  Perplexity does live up to their name.  I am perplexed by this

**Comment by felixfrank** [comment]
I have the same issue, they seem to have deactivated the promo code.

**Comment by didoslavov** [comment]
Same issue here

**Comment by cliftonburt** [comment]
samesies

**Comment by dym-ok** [comment]
The new code is `FREEPRONH9HQOG2` apparently.

**Reply by didoslavov** [reply]
Not working for me.

**Reply by Nathan-Bransby-NMT** [reply]
Invalid still

**Reply by dym-ok** [reply]
Sorry guys, email said it's a disposable personal one. I got it in the email about the second GitHub Universe 2025 day.
